alerting/src/main/kotlin/org/opensearch/alerting/resthandler/RestGetWorkflowAlertsAction.kt

```diff
@@ -55,11 +55,16 @@ class RestGetWorkflowAlertsAction : BaseRestHandler() {
         val severityLevel = request.param("severityLevel", "ALL")
         val alertState = request.param("alertState", "ALL")
         val workflowId: String? = request.param("workflowIds")
+        val alertId: String? = request.param("alertIds")
         val getAssociatedAlerts: Boolean = request.param("getAssociatedAlerts", "false").toBoolean()
         val workflowIds = mutableListOf<String>()
         if (workflowId.isNullOrEmpty() == false) {
             workflowIds.add(workflowId)
         }
+        val alertIds = mutableListOf<String>()
+        if (alertId.isNullOrEmpty() == false) {
```

Comment:
nitpick: use `isNotEmpty()` or `isNotBlank()`

Reply:
alertId is of type String?

isNotEmpty() and isNotBlank() can be used only if it is not nullable. so the isNullOrEmpty() check takes care of it. There isn't a negated implementation of isNullOrEmpty